client/src/client.rs

```diff
@@ -908,6 +908,77 @@ pub trait RpcApi: Sized {
         )
     }
 
+    /// Attempts to add or remove a node from the addnode list.
+    /// Or try a connection to a node once.
```

Comment:
I think this documentation is wrong. `fn add_node()` only adds. `fn remove_node()` and `fn onetry_node()` are missing documentation.

Reply:
Thanks, fixed the documentation with https://github.com/rust-bitcoin/rust-bitcoincore-rpc/pull/171/commits/8b92fa3f82a300f3291b87a0e079d8266dd94dd2